homeassistant/components/switcher_kis/cover.py

```diff
@@ -92,32 +90,6 @@ def _update_data(self) -> None:
             data.direction[self._cover_id] == ShutterDirection.SHUTTER_UP
         )
 
-    async def _async_call_api(self, api: str, *args: Any) -> None:
-        """Call Switcher API."""
-        _LOGGER.debug("Calling api for %s, api: '%s', args: %s", self.name, api, args)
```

Comment:
The `_LOGGER` is not used anymore in this file, please remove it

Reply:
it is done.